tests/fips/gnutls/gnutls_base_check.pm

```diff
@@ -1,24 +1,36 @@
-# Copyright 2020 SUSE LLC
+# Copyright 2020-2021 SUSE LLC
 # SPDX-License-Identifier: GPL-2.0-or-later
 #
-# Package: gnutls openssl
-# Summary: SLES15SP2 FIPS certification, we need to certify gnutls and libnettle
+# Package: gnutls / libnettle
+# Summary: SLES15SP2 and SLES15SP4 FIPS certification need to certify gnutls and libnettle
 #          In this case, will do some base check for gnutls
-# Maintainer: rfan1 <richard.fan@suse.com>
-# Tags: poo#63223, tc#1744099
+#
+# Maintainer: rfan1 <richard.fan@suse.com>, Ben Chou <bchou@suse.com>
+# Tags: poo#63223, poo#102770, tc#1744099
 
 use base 'consoletest';
 use testapi;
 use strict;
 use warnings;
 use utils 'zypper_call';
-use version_utils qw(is_tumbleweed);
+use version_utils qw(is_tumbleweed is_sle);
```

Comment:
is_tumbleweed is not used, right? then we can remove

Reply:
Good question, I think TW also apply this case. But I think my adjustment will not affect the TW test. So I think we should keep it there.